File: 10-is-palindrome/is_palindrome.py

def is_palindrome(word):

    string_list=[letter for letter in word]
    reverse_list=string_list[::-1]
    reverse_word=''.join(reverse_list)
    
    if reverse_word==word:
        return True
    else:
        return False

print(is_palindrome('engage'))

File: 10-is-palindrome/is-palindrome.js
const isPalindrome=(str)=>{

    const reverseString=[]

    for(var i=str.length-1; i>=0; i--){
        reverseString.push(str[i])
    }

    if(reverseString.join('')==str){
        return true
    }else{
        return false
    }

}

console.log(isPalindrome('engage'))